internal/libsndfile/v1.1.0.patch

```diff
@@ -0,0 +1,30 @@
+--- a/src/ogg_opus.c	Sun Mar 27 07:48:57 2022
++++ b/src/ogg_opus.c	Fri Jul  8 16:08:35 2022
+@@ -162,8 +162,8 @@
+ #if HAVE_EXTERNAL_XIPH_LIBS
+
+ #include <ogg/ogg.h>
+-#include <opus/opus.h>
+-#include <opus/opus_multistream.h>
++#include <opus.h>
++#include <opus_multistream.h>
+
+ #include "ogg.h"
+ #include "ogg_vcomment.h"
+--- a/CMakeLists.txt	Sun Mar 27 07:48:57 2022
++++ b/CMakeLists.txt	Fri Jul  8 17:09:19 2022
+@@ -406,10 +406,10 @@
+ target_link_libraries (sndfile
+ 	PRIVATE
+ 		$<$<BOOL:${LIBM_REQUIRED}>:m>
+-		$<$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>:Vorbis::vorbisenc>
+-		$<$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>:FLAC::FLAC>
++		$<$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>:$<TARGET_LINKER_FILE:vorbisenc>>
++		$<$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>:$<TARGET_LINKER_FILE:FLAC>>
+ 		$<$<AND:$<BOOL:${ENABLE_EXPERIMENTAL}>,$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>,$<BOOL:${HAVE_SPEEX}>>:Speex::Speex>
+-		$<$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>:Opus::opus>
++		$<$<BOOL:${HAVE_EXTERNAL_XIPH_LIBS}>:$<TARGET_LINKER_FILE:opus>>
+ 		$<$<BOOL:${HAVE_MPEG}>:MPG123::libmpg123>
+ 		$<$<BOOL:${HAVE_MPEG}>:Lame::Lame>
+ 	)
```

Comment:
*bad dalek impression*

EXPLAIN?

Reply:
1) libsndfile thinks that the Opus lib being provided is in an _installed_ location :D so we patch this to be explicitly compatible with the built version...cuz we build it lol.

2) Originally libsndfile wanted `vorbisenc.obj` which wasn't being provided by the `vorbisenc` target, so I patched it to specifically used whatever library is provided by the target instead of assuming..it lead to similar breakages so I changed the ones we're concerned with.